src/screens/signup/steps/IdStep.tsx

```
@@ -0,0 +1,84 @@
+  }, [value])
+
+  
+  const handleRecommand = () => {
+    onChange(recommandId)
+  }
+
+  return(
+    <Continaer>
+      <ContentLayout>
+        <Typography children='이메일 또는 아이디 생성' font='medium28' color='defaultBlack' />
+        <AuthInput label='회원가입하고 책마루에 가입하세요!' placeholder='이메일 또는 아이디 생성' isError={false} warningMessage="이미 존재하는 아이디입니다." value={value} onChangeText={onChange} />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)